src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -85,7 +85,10 @@ public function load($blnNoCache=false)
 					include $file;
 				}
 			}
-			catch (\InvalidArgumentException $e) {}
+			catch (\InvalidArgumentException $e)
+			{
+				\System::getContainer()->get('monolog.logger.contao')->notice($e->getMessage(), ['exception' => $e]);
```

Comment:
> The PaletteNotFoundException class is only added in this PR, therefore there cannot be a BC break.

But previously it threw an `InvalidArgumentException`, so we must stay compatible with that if someone already catched it.